.github/workflows/checks.yml

```diff
@@ -0,0 +1,18 @@
+name: checks
+
+on: [push, pull_request]
+
+jobs:
+  checks: 
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install python 
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.9'
+      - name: flake8 action
```

Comment:
Since there wouldn't be a lot of customization in the usage of the commands, I thought of using third-party actions but I now see that I was wrong, I will install and use them directly.